examples/build_examples.sh

```diff
@@ -18,6 +18,7 @@ EXAMPLES=(
   "${EXAMPLES_DIR}/resource/ferris"
   "${EXAMPLES_DIR}/resource/ferris_says_hello"
   "${EXAMPLES_DIR}/resource/hello_message"
+  "./tests/test_container"
```

Comment:
The test container should not be built from the examples script. Use a `build.rs` in the test crate and add `sextant` to the `dev-dependencies` of `test`. Think the `build.rs` should just package. Invoking a `cargo build` from a `build.rs` feels odd.

Reply:
I think a `build.rs` is not the way to go here as the packaging has to occur _after_ the build is done. I will add a bash script similar to the one of `build_examples.sh`.

Reply:
ok